sites/website/src/docs/design-systems/using-the-design-system.md

```diff
@@ -0,0 +1,130 @@
+---
+id: using-the-design-system
+title: Using the FAST Design System 
+sidebar_label: Using the Design System
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/design-systems/using-the-design-system.md
+---
+
+This section goes into further detail on how the Design System can be used. It will use [FAST Frame](/docs/design-systems/fast-frame) Design System in the examples but none are *specific* to [FAST Frame](/docs/design-systems/fast-frame); use these features with your own Design System if you're not using [FAST Frame](/docs/design-systems/fast-frame).
+
+### Setting Design System Properties
+Design System properties can always be set by simple property assignment in JavaScript. Simply obtain a reference to the Custom Element instance and assign the property.
+
+**Example: Setting a Design System value by property assignment**
+```ts
+const provider = document.querySelector('fast-design-system-provider');
+provider.designUnit = 6; // Set the Design System property.
+```
+
+or with FAST Element's [property bindings](/docs/fast-element/declaring-templates#properties):
+
+```ts
+import { html } from "@microsoft/fast-element";
+
+export const template = html`
+<fast-design-system-provider :designUnit="${() => 6}">
+</fast-design-system-provider>
+`
+```
+
+*Most* Design System properties can be set by setting an HTML *attribute* on the element instance. The attribute name for the Design System property will be the corresponding JavaScript property name by default, but it can be customized or disabled for each Design System property. See [Declaring the Design System](/docs/design-systems/creating-a-design-system#declaring-the-design-system) for more info.
+
+**Example: Setting a Design System value by HTML attribute**
+```html
+<fast-design-system-provider design-unit="6">
+
+</fast-design-system-provider>
+```
+
+## `use-defaults`
+Every [DesignSystemProvider](/docs/api/fast-foundation.designsystemprovider) supports the `use-defaults` boolean attribute. `use-defaults` is a mechanism to assign the [default Design System values](/docs/design-systems/creating-a-design-system#default) to the [DesignSystemProvider](/docs/api/fast-foundation.designsystemprovider).
+
+It is generally a good idea to have a [DesignSystemProvider](/docs/api/fast-foundation.designsystemprovider) with the `use-defaults` attribute wrapping all of the app UI; this will ensure all properties used by descendent components are defined. This is especially important if the Design System properties reflect to CSS custom properties because the CSS custom properties are only created when the value is set.
```

Comment:
omit "generally"

Reply:
replace "wrapping all of the app UI; this will ensure" with "wrapping all of the app UI to ensure"